driver.sh

```diff
@@ -141,6 +141,11 @@ check_dependencies() {
   command -v timeout
   command -v unbuffer
 
+  for readelf in llvm-readelf-9 llvm-readelf-8 llvm-readelf-7 llvm-readelf readelf; do
```

Comment:
thanks for the addition of `llvm-readelf`.  Can you additionally remove `readelf`?  I'd like to get to the point where we can build in an environment that contains no GNU binutils.  If `llvm-readelf` fails for whatever reason, than that's a bug we should fix.

Reply:
`driver.sh` should be environment agnostic. I'm fine with making that change but if `llvm-readelf` isn't installed, we need to avoid trying to call it so I'll need to rework that.

Reply:
Maybe it is worth adding an environment variable that could be exported/set to disable GNU fallbacks like `NO_GNU`.

Reply:
The ultimate goal needs to be that we can build with no GNU tools installed.  That way if any of the kernel build steps are not hermetic (ex. they don't invoke they linker they were told to) they fail hard.  That's a feature, not a bug.

Reply:
Yes, I agree. If that is the direction you want to take it now, that's fine with me but that means that `driver.sh` will be exclusively tied to our CI setup because we can't expect people to remove their copy of binutils to use it.

Otherwise, the alternative is to make `env-setup.sh` either remove or check for the absence of GNU packages (or avoiding installing them during the docker build).

Reply:
Those are all good ideas and good points.  We can cross that bridge when we get there (explicitly checking that the env is does not have GNU binutils).  I think for this, expecting llvm-readelf to be installed (and lld and llvm-ar and llvm-objcopy etc) is not too demanding.

Reply:
Indeed. Most distributions will include the LLVM tools when installing clang (or make them very easy to get otherwise).

I have removed `readelf` from the list so that the build fails when `llvm-readelf` isn't found.